xbmc/guilib/listproviders/DirectoryProvider.cpp

```diff
@@ -47,6 +48,7 @@
 
 using namespace XFILE;
 using namespace KODI::MESSAGING;
+using namespace KODI::VIDEO;
```

Comment:
Typically I would do:

```c++
using namespace KODI;
using namespace MESSAGING;
using namespace VIDEO;
```

because ultimately everything should be moved into the KODI namespace. However, outside of the video code (like here), I would also omit the `VIDEO` namespace and prefix all video calls below with VIDEO:: like this:

```c++
if (VIDEO::IsVideo(*item))
```

The first namespace preference thing feel free to ignore, but what do you think about not having `using namespace VIDEO` outside of most video code?

Reply:
I try to do as the romans but the romans can't make up their mind. 

My personal view is aligned with yours, ie, all code should be in Kodi namespace (notice, no screaming) and then you refer relative to that elsewhere. No using namespace clauses.

Reply:
I have added a fixup-commit that does this where it is possible. Tell me if you prefer the consistency or "sanity".